### ISSUE TITLE
ionic 2 app with Bluetooth keyboard on ios

### ISSUE THREAD

**Issue by nsandeep87** [ionic-team/ionic#12246]
I am not able to use Bluetooth keyboard with ionic input in the ionic2 app. Each time I try to set focus on the inputbox, it goes out of scope. However it works fine when the ipad is not paired to the keyboard.

Environment details:
ionic-angular: 2.0.1
@angular/ - 2.2.1
ios version - 10.x

**Comment by AmitMY** [comment]
- Can you please fill the issue template fully.
- I recommend updating to 3.5.0, as there were many input and keyboard changes. It is actually super simple to upgrade, and will take around 5-10 minutes. See [guide](https://forum.ionicframework.com/t/guide-how-to-update-to-ionic-3-x/87516).

**Comment by haninaguib** [comment]
Having the same issue. Here is my ionic info
global packages:

    @ionic/cli-utils : 1.4.0
    Cordova CLI      : 6.5.0 
    Ionic CLI        : 3.4.0

local packages:

    @ionic/app-scripts              : 1.3.12
    @ionic/cli-plugin-cordova       : 1.4.0
    @ionic/cli-plugin-ionic-angular : 1.3.1
    Cordova Platforms               : browser 4.1.0 ios 4.3.1
    Ionic Framework                 : ionic-angular 3.5.0

System:

    Node       : v7.4.0
    OS         : macOS Sierra
    Xcode      : Xcode 8.3.3 Build version 8E3004b 
    ios-deploy : 1.9.1 
    ios-sim    : 5.0.13 
    npm        : 4.0.5

**Comment by nsandeep87** [comment]
Yes, just verified that the issue is caused because of the plugin ionic-plugin-keyboard(2.2.1). If I remove this plugin(ionic plugin rm ionic-plugin-keyboard), the app works fine. So can anyone please let me know if they have any workaround.

**Comment by jgw96** [comment]
Hello everyone! I am going to consider this as a feature request as Ionic was not originally built with the `using a bluetooth keyboard with an ipad` use case in mind. I'm going to move this issue over to our internal list of feature requests for evaluation. We are continually prioritizing all requests that we receive with outstanding issues. We are extremely grateful for your feedback, but it may not always be our next priority. I'll copy the issue back to this repository when we have begun implementation. Thanks!

**Comment by jgw96** [comment]
This issue was moved to ionic-team/ionic-feature-requests#265

**Comment by schie** [comment]
Hello, @jgw96!

Has there been any movement on this issue?

**Comment by amie-wilt** [comment]
@jgw96, any movement on this?

@schie, have you found anything on this? Thanks ahead of time.

**Comment by lincolnthree** [comment]
Hey @jgw96 !

I just ran into this. Seems like a to me TBH. Using a bluetooth keyboard is very common with iOS devices, and the behavior is definitely a bit crazy. I'd expect a bluetooth keyboard to work just like a native one. But obviously I'm not in your shoes, all I know is that users are complaining :)

I also noticed that the feature request no longer exists --- is this being tracked somewhere else?
